resources/views/contests/index.blade.php

```diff
@@ -31,10 +31,12 @@
                             </tr>
                         </thead>
                         <tbody>
+
+
                             @foreach($contests as $contest)
                                 <tr class="table__row">
                                     <td class="table__cell"><a href="{{ route('contests.show', $contest->id) }}">{{ $contest->id }}</a></td>
-                                    <td class="table__cell">{{ $contest->campaign_id }}</td>
+                                    <td class="table__cell"> {{ link_to(config('services.phoenix.uri') .'/node/' . $contest->campaign_id, $contest->campaign_id) }} </td>
```

Comment:
Yeah I think that'd make sense (and be consistent with our usage of `route()` elsewhere), especially if we're still considering removing the HTML package from this app at some point.